pxtblocks/blocklyloader.ts

```diff
@@ -676,7 +676,7 @@ namespace pxt.blocks {
     function initContextMenu() {
         // Translate the context menu for blocks.
         let msg: any = Blockly.Msg;
-        msg.DUPLICATE_BLOCK = lf("Duplicate");
+        msg.DUPLICATE_BLOCK = lf("Duplicate{id:block}");
```

Comment:
OK, if you like it.

The reason I choose "duplicate block" was that it's natural word order of English.